src/viam/robot/client.py

```diff
@@ -741,12 +759,15 @@ async def discover_components(
         queries: List[DiscoveryQuery],
     ) -> List[Discovery]:
         """
-        Get the list of discovered component configurations.
+        Get a list of discovered potential component configurations for webcam cameras, for example listing different supported resolutions.
```

Comment:
Let's go with something like this. As anything does get implemented we'll have to update it. But I can't think of usecases for it. For webcams it is because it suggests relevant resolutions. I can't think of a similar thing for other components
```suggestion
        Get a list of discovered potential component configurations, for example listing different supported resolutions. Currently only works for some cameras.
```